src/pages/ReportDetailsPage.tsx

```diff
@@ -226,14 +226,17 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         Report.leaveRoom(report.reportID, isWorkspaceMemberLeavingWorkspaceRoom);
     }, [isPolicyEmployee, isPolicyExpenseChat, isRootGroupChat, report.reportID, report.visibility]);
 
+    const [moneyRequestReportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${moneyRequestReport?.reportID}`);
```

Comment:
Are we 100% sure we should use `moneyRequestReport` and not `report` here? I remember there was a blocker on a recent Unapproved PR b/c we used `moneyRequestReport` where we should have used `report` :D

Reply:
@Beamanator the report that we're **unapproving** is `moneyRequestReport`. And the exported to accounting action takes place inside it too. So I'm confident that this is correct

Reply:
Then let's get this merged!!